pymysql/tests/test_issues.py

```diff
@@ -381,3 +383,30 @@ def test_issue_175(self):
                     warnings.filterwarnings("ignore")
                     cur.execute('drop table if exists test_field_count')
 
+    def test_pull_request_for_fix_tuple_and_unicode_encoding_issue(self):
+        """ Encoding error appears when iterable is passed in query args list.
```

Comment:
Too long name. "pull_request_for_fix" is too verbose.
Maybe, "test_encoding_tuple_of_unicode_and_bytes"?